packages/bootstrap/scss/listview/_theme.scss

```diff
@@ -1,9 +1,16 @@
-@include exports('listview/theme') {
-    .k-listview > .k-state-focused {
-        box-shadow: inset 0 0 0 3px try-darken( $node-bg, 7 );
+@include exports( "listview/theme" ) {
```

Comment:
This file is identical to the default theme file, so I'd suggest importing the default theme, in order to reduce code duplication.

Reply:
Done